packages/ui/src/components/Visualization/ContextToolbar/FlowExportImage/HiddenCanvas.tsx

```diff
@@ -0,0 +1,160 @@
+import { action, Model, VisualizationProvider, VisualizationSurface } from '@patternfly/react-topology';
+import { FunctionComponent, useContext, useEffect, useMemo, useRef, useState } from 'react';
+
+import { BaseVisualCamelEntity } from '../../../../models/visualization/base-visual-entity';
+import { VisibleFlowsContext } from '../../../../providers/visible-flows.provider';
+import { CanvasEdge, CanvasNode, LayoutType } from '../../Canvas/canvas.models';
+import { ControllerService } from '../../Canvas/controller.service';
+import { FlowService } from '../../Canvas/flow.service';
+
+interface HiddenCanvasProps {
+  entities: BaseVisualCamelEntity[];
+  layout: LayoutType;
+  onReady: (element: HTMLElement) => void;
+}
+
+export const HiddenCanvas: FunctionComponent<HiddenCanvasProps> = (props) => {
```

Comment:
```suggestion
export const HiddenCanvas: FunctionComponent<HiddenCanvasProps> = ({
  entities,
  layout = LayoutType.DagreHorizontal,
  onReady,
}) => {
```

Reply:
If we allow `undefined` for layout, we can set it to `Horizontal` here; the advantage is that any other consumer will have this logic out of the box.

Another consumer of this component might be the "Generate Route Documentation" button.

<img width="365" height="110" alt="Image" src="https://github.com/user-attachments/assets/4e1aed7a-27e0-4932-b97e-79b9eb4e04b8" />